### ISSUE TITLE
mtime: undefined method `mtime' for #<File:0x1d0>

### ISSUE THREAD

**Issue by cmoulliard** [asciidoctor/asciidoctor.js#203]
This syntax 

```
function convertToHtml3(f) {
    var attrs = opal.hash({'showtitle'     : '',
        'stylesheet'    : 'foundation',
        'stylesdir'     : '/Users/chmoulli/Code/github/asciidoctor/asciidoctor-stylesheet-factory/stylesheets'});
    var options = opal.hash({doctype    : 'article',
        attributes : attrs});
    try {
        processor.$convert_file(f, options);
    } catch (e) {
        console.error(e.stack);
    }
}
```

generates this error 

```
mtime: undefined method `mtime' for #<File:0x1d0>
    at SingletonClass_alloc.$new (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/node_modules/opal-npm-wrapper/index.js:4512:15)
    at $File.Opal.defn.TMP_1 (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/node_modules/opal-npm-wrapper/index.js:3075:54)
    at $File.method_missing_stub [as $mtime] (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/node_modules/opal-npm-wrapper/index.js:904:35)
    at module_constructor.$load (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.min.js:641:95)
    at module_constructor.<anonymous> (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.min.js:645:369)
    at module_constructor.<anonymous> (/Users/chmoulli/.nvm/versions/node/v4.2.0/lib/node_modules/asciidoctor.js/dist/npm/asciidoctor-core.min.js:652:85)
    at convertToHtml2 (/Users/chmoulli/Google-Drive/REDHAT/RH-GP/Reports/2016/asciidoc.js:57:30)
    at readFile (/Users/chmoulli/Google-Drive/REDHAT/RH-GP/Reports/2016/asciidoc.js:24:5)

```

**Comment by mojavelinux** [comment]
Looks like we need to implement mtime on File, @Mogztter.

**Comment by Mogztter** [comment]
I've already implemented `mtime`  in Opal 0.10+.
This issue will be fixed in the next release of Asciidoctor.js when https://github.com/asciidoctor/asciidoctor.js/pull/199 is merged.

**Comment by mojavelinux** [comment]
Oops, that's right! And I even gave it a :tada:. Where is my brain?

**Comment by Mogztter** [comment]
> Oops, that's right! And I even gave it a :tada:. Where is my brain?

Ha ha :laughing:

Once everything is merged, I will most likely do a "preview release" (`1.5.5-2) because I've recently enabled a lot of I/O functions when running on Node.js.

Thanks again @cmoulliard for pushing this project forward :wink:
 I will try my best to keep it up!

**Comment by Mogztter** [comment]
Resolved by #199
